app/providers/atmosphere/local_pdp.rb

```diff
@@ -0,0 +1,54 @@
+#
+# Local PDP with basic service sharing functionality.
+# Does not require external services to operate.
+#
+module Atmosphere
+  class LocalPdp
+    def initialize(current_user)
+      @current_user = current_user
+    end
+
+    def can_start_in_production?(at)
+      Rails.logger.debug("can_start_in_production: using local PDP")
+      uat = UserApplianceType.where(
+        user: @current_user,
+        appliance_type: at
+      )
+      uat.present?
+    end
+
+    def can_start_in_development?(at)
+      Rails.logger.debug("can_start_in_development: using local PDP")
+      uat = UserApplianceType.where(
+          user: @current_user,
+          appliance_type: at
+      )
+      uat.present? and ['developer', 'manager'].include? uat.first.role
```

Comment:
Don't use `and`, instead use `&&` (http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby)